plugins/bigquery/dbt/adapters/bigquery/connections.py

```diff
@@ -458,26 +458,32 @@ def copy_bq_table(self, source, destination, write_disposition):
         conn = self.get_thread_connection()
         client = conn.handle
 
-        source_ref = self.table_ref(
-            source.database, source.schema, source.table, conn)
+        if type(source) is not list:
+            source = [source]
+
+        source_ref_array = [self.table_ref(
+            src_table.database, src_table.schema, src_table.table, conn)
+            for src_table in source]
```

Comment:
Appreciate the backwards compatibility here! Could you add a comment to the effect of, `source` used to expect only one table reference, but now we accept multiple?

Reply:
Added